pkg/datastore/postgres/repositories/address_repository_test.go

```diff
@@ -0,0 +1,169 @@
+// VulcanizeDB
+// Copyright © 2019 Vulcanize
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package repositories_test
+
+import (
+	"github.com/jmoiron/sqlx"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/vulcanize/vulcanizedb/pkg/datastore/postgres"
+	"github.com/vulcanize/vulcanizedb/pkg/datastore/postgres/repositories"
+	"github.com/vulcanize/vulcanizedb/pkg/fakes"
+	"github.com/vulcanize/vulcanizedb/test_config"
+	"strings"
+)
+
+var _ = Describe("address lookup", func() {
+	var (
+		db      *postgres.DB
+		repo    repositories.AddressRepository
+		address = fakes.FakeAddress.Hex()
+	)
+	BeforeEach(func() {
+		db = test_config.NewTestDB(test_config.NewTestNode())
+		test_config.CleanTestDB(db)
+		repo = repositories.AddressRepository{}
+	})
```

Comment:
Yeah I think it would be better to have the `test_config.CleanTestDB(db)` in an `AfterEach` statement, I might be missing something but I think with the way it is right now any addresses that are added in the last test to be ran will be left over after this suite exits and could potentially lead to issues down the road with other tests.

Reply:
I'm actually a bit unclear about when we need to do `db.Close` and when we don't. I thought that `db.Close` was something that we were doing in the `mcd_transformers` repo since we're building up and tearing down the db for each test run, but I could be wrong.

I created VDB-799 as a way to make sure we're handling test cleanup consistently, does that make sense to handle this as part of that story? In the mean time, I'll add `test_config.CleanTestDB(db)` to an `AfterEach`.